build/source/engine/canopySnow.f90

```diff
@@ -123,10 +132,14 @@ subroutine canopySnow(&
 
  ! model prognostic variables (input/output)
  scalarCanopyIce           => prog_data%var(iLookPROG%scalarCanopyIce)%dat(1),             & ! intent(inout): [dp] mass of ice on the vegetation canopy (kg m-2)
+ scalarCanairTemp          => prog_data%var(iLookPROG%scalarCanairTemp)%dat(1),             & ! intent(input): [dp] temperature of the canopy air space (k)
 
  ! model fluxes (input)
  scalarSnowfall            => flux_data%var(iLookFLUX%scalarSnowfall)%dat(1),              & ! intent(in): [dp] computed snowfall rate (kg m-2 s-1)
  scalarCanopyLiqDrainage   => flux_data%var(iLookFLUX%scalarCanopyLiqDrainage)%dat(1),     & ! intent(in): [dp] liquid drainage from the vegetation canopy (kg m-2 s-1)
+ scalarWindspdCanopyTop    => flux_data%var(iLookFLUX%scalarWindspdCanopyTop)%dat(1),      & ! intent(in): [dp] windspeed at the top of the canopy (m s-1)
+ scalarWindspdCanopyBottom => flux_data%var(iLookFlux%scalarWindspdCanopyBottom)%dat(1),   & ! intent(in): [dp] windspeed at the height of the bottom of the canopy (m s-1)
```

Comment:
drop the `height of the` to be consistent with the previous comment (previous line)